lua/wire/server/wirelib.lua

```diff
@@ -622,18 +629,22 @@ end
 function WireLib.TriggerOutput(ent, oname, value, iter)
 	if not IsValid(ent) then return end
 	if not HasPorts(ent) then return end
-	if (not ent.Outputs) then return end
 
-	local output = ent.Outputs[oname]
+	local entTbl = ent:GetTable()
+	local outputs = entTbl.Outputs
+	if not outputs then return end
+
+	local output = outputs[oname]
 	if not output then return end
 
-	local ty = WireLib.DT[output.Type]
+	local outputType = output.Type
```

Comment:
I don't think every single table index needs a local. Too many locals makes a mess.

Reply:
I tried to localize only the values that get reused at least a couple times to avoid having to get them again. I'll remove the lesser used ones if it's preferable, though.